e2e/bitrise.yml

```diff
@@ -0,0 +1,11 @@
+format_version: 11
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  test_install_unity:
+    steps:
+    - path::./:
+        title: Install unity test
+        inputs:
+        - unity_version: 2022.3.1f1
+        - unity_version_changeset: fb119bb0b476
```

Comment:
nit: Can we verify the result of the step run somehow? We usually do things like `ls`ing an expected directory.

Reply:
We can't check with the CLI. We can check a file `/Applications/Unity/Hub/Editor/2022.3.1f1/modules.json` which contains the all information but it's not a stable solution because unity can change the structure of the file and the path as well.